gp-includes/install-upgrade.php

```diff
@@ -6,12 +6,26 @@
  * @since 1.0.0
  */
 
+/**
+ * Tells whether the database needs to be installed/upgraded or not.
+ *
+ * @since 4.0.0
+ */
+function gp_db_upgrade_required(): bool {
```

Comment:
As this is a boolean function, who replies to the question "do we need to upgrade the database", I suggest adding the `is_` in the function name and in its usages:

```suggestion
function is_gp_db_upgrade_required(): bool {
```